src/ecephys_etl/modules/vbn_create_stimulus_table/create_stim_table.py

```diff
@@ -834,7 +834,7 @@ def compute_vbn_block_frame_times(
             pd_interval_duration / (num_vsyncs_per_diode_toggle + total_delay)
         )
 
-        if total_delay > 0:
+        if abs(total_delay) > 0:
```

Comment:
Does the code in this now modified `if` block actually deal correctly with total delay that is negative?

Reply:
yes - Corbett is happy with the results from this. From https://github.com/AllenInstitute/ecephys_etl_pipelines/issues/25

Reply:
> We basically just need to edit line 830:

> if total_delay > 0:

> becomes

> if abs(total_delay)>0:

> And that should fix it! I’ll add more info to the issue tomorrow, but I think the basics are sketched out there.

> Cheers,
> c